src/app/components/navbar/navbar.component.scss

```diff
@@ -0,0 +1,221 @@
+            background: #f1f3f4;
+            border: 1px solid transparent;
+            -webkit-border-radius: 8px;
+            border-radius: 8px;
+            margin-left: auto;
+            margin-right: auto;
+            max-width: 720px;
+            .input {
+              background: transparent;
+              border: none;
+              font: normal 16px sans-serif;
+              -webkit-font-variant-ligatures: none;
+              font-variant-ligatures: none;
+              height: 46px;
+              outline: none;
+              width: 100%;
+              box-sizing: border-box;
+              margin-left: 56px;
+              margin-right: 49px;
+              &:focus-visible {
+                outline-width: 0;
+              }
+
+              &::placeholder {
+
+                font-weight: 100;
+              }
+            }
+          }
+
+          .btn {
+            width: 56px;
+            height: 46px;
+            background: none;
+            border: none;
+            position: absolute;
+            top: 0;
+            right: 0;
+
+            &.left {
+              left: 0;
+            }
+          }
+
+          .pic {
+            padding: 8px;
+          }
+
+
+        }
+      }
+
+      .btns-container {
+        display: flex;
+
+        .refresh {
+          @extend %icons;
+          background-image: url(/assets/images/svg/refresh.svg);
+        }
+
+        .view {
+          @extend %icons;
+          background-image: url(/assets/images/svg/view.svg);
+        }
+
+        .settings {
+          @extend %icons;
+          background-image: url(/assets/images/svg/settings.svg);
+        }
+
+        .search {
+          @extend %icons;
+          background-image: url(/assets/images/svg/search.svg);
+          opacity: 1 !important;
+          display: none;
+
+          @media (max-width: 794px) {
+            display: block;
+          }
+        }
+      }
+    }
+
+    // third col
+    .th-col {
+      display: flex;
+      align-items: center;
+      height: 48px;
+      padding: 0 4px 0 28px;
+
+      @media (max-width:405px) {
+        display: none;
+      }
+
+      .apps {
+        @extend %icons;
+        background-image: url(/assets/images/svg/apps.svg);
+        position: relative;
+        right: 2px;
+      }
+
+      .user-pic-container {
+        padding: 4px 6px;
+        display: inline-block;
+
+        .user-pic {
+          border-radius: 50%;
+          height: 32px;
+          width: 32px;
+          margin-bottom: 1px;
+        }
+      }
+    }
+  }
+}
+
+%icons {
+  height: 48px;
+  width: 48px;
+  background-size: 24px 24px;
+  background-repeat: no-repeat;
+  background-position: center;
+  opacity: 1;
+  transition: opacity 0.218s linear;
+  position: relative;
+
+  &:hover {
+    opacity: 0.87;
+  }
+
+  //&.pop::before {
+  //  top: 50px
+  //}
+}
+
+//.disabled {
+//  cursor: not-allowed !important;
+//  opacity: 0.25 !important;
+//}
```

Comment:
Rizwan bhai i am changing px to rem/em in all css files
and add commented code in ToDo